include/xrpl/protocol/STTx.h

```diff
@@ -102,6 +102,10 @@ class STTx final : public STObject, public CountedObject<STTx>
     SeqProxy
     getSeqProxy() const;
 
+    /** Returns the first non-zero value of (Sequence, TicketSequence). */
+    std::uint32_t
+    getSeqValue() const;
```

Comment:
fyi,  the [account permission](https://github.com/XRPLF/rippled/pull/5164) feature changes how sequence work. it introduces another wrapper class STTxDelegated that's used to fetch the sequence